### PR TITLE
Redraft of all eps

### PR DESCRIPTION
**Pull request by bobturneruk**

### CODE REVIEW

File: _episodes/01-getting-started-with-conda.md [hunk 1]
```diff
@@ -37,13 +37,13 @@ When working with a programming language, such as Python, that can do almost _an
 
 ### Dependencies
 
-A bit further into your programming career you may notice/have noticed that many packages do not just do everything on their own. Instead, they _depend_ on other packages for their functionality. For example, the [Scipy package](https://www.scipy.org/about.html) is used for numerical routines. To not reinvent the wheel, the package makes use of other packages, such as numpy (numerical python) and matplotlib (plotting) and many more. So we say that numpy and matplotlib are *dependencies* of Scipy. 
+A bit further into your programming career you may notice/have noticed that many packages do not just do everything on their own. Instead, they _depend_ on other packages for their functionality. For example, the [`Scipy` package](https://www.scipy.org/about.html) is used for numerical routines. To not reinvent the wheel, the package makes use of other packages, such as `numpy` (numerical python) and `matplotlib` (plotting) and many more. So we say that `numpy` and `matplotlib` are *dependencies* of `Scipy`.
```
Comment (ns-rse) [inline _episodes/01-getting-started-with-conda.md]: I realise its inherited but perhaps lower-case `scipy` to be consistent with `numpy` and `matplotlib`.

File: _episodes/02-working-with-environments.md [hunk 2]
```diff
@@ -347,145 +347,6 @@ Running `ls` (linux) / `dir` (Windows) on your anaconda `envs/` directory will l
 > previously installed Conda environments.
 {: .callout}
 
-
```
Comment (ns-rse) [inline _episodes/02-working-with-environments.md:350]: Agree with removing this, its an introductory course so I think its appropriate to keep it simple and have one central place where virtual environments are created and stored, its like teaching people to run before they can walk recommending project specific environments.

Not wanting to denigrate people's experience/knowledge but based on previous experience there is always a proportion who don't understand paths and directory structure as well as might be required and this section has potential to confuse.

File: _episodes/02-working-with-environments.md [hunk 3]
```diff
@@ -486,35 +486,6 @@ For more on modifying your `.condarc` file, see the
 > {: .solution}
 {: .challenge}
 
-
```
Comment (ns-rse) [inline _episodes/02-working-with-environments.md:489]: Yep, keep it focused on Python, most R users should be learning about `renv` (in my view).

File: _episodes/02-working-with-environments.md [hunk 4]
```diff
@@ -101,13 +101,13 @@ $ conda create --name python36-env python=3.6
 ~~~
 {: .language-bash}
 
-> ## Always specify a version number for each package you wish to install
+> ## You can specify a version number for each package you wish to install
 >
 > In order to make your results more reproducible and to make it easier for research colleagues to 
-> recreate your Conda environments on their machines it is a "best practice" to always explicitly 
+> recreate your Conda environments on their machines it is sometimes a good practice to always explicitly
```
Comment (ns-rse) [inline _episodes/02-working-with-environments.md]: As discussed I agree explicit versioning can cause headaches if another package has pegged the version it depends on.

File: _episodes/04-sharing-environments.md [hunk 5]
```diff
@@ -348,74 +348,5 @@ from the environment.
 > (after creating the Conda environment and installing all Conda installable dependencies).
 {: .callout}
 
-## Making Jupyter aware of your Conda environments
```
Comment (ns-rse) [inline _episodes/04-sharing-environments.md:361]: Agree this is a fairly advanced level that could cause confusion.

File: _episodes/01-getting-started-with-conda.md [hunk 6]
```diff
@@ -47,6 +47,14 @@ When starting with programming we may not use many packages yet and the installa
 Nowadays there are several environment management systems following a similar idea:
 Instead of having to use multiple computers or virtual machines to run different versions of the same package, you can install packages in isolated environments.
 
+> ## Virtual Machines
```
Comment (ns-rse) [inline _episodes/01-getting-started-with-conda.md:50]: :+1: useful to explain concepts that are likely to be unfamiliar.

File: _episodes/02-working-with-environments.md [hunk 7]
```diff
@@ -103,10 +103,18 @@ $ conda create --name python36-env python=3.6
 > ## You can specify a version number for each package you wish to install
 >
 > In order to make your results more reproducible and to make it easier for research colleagues to 
-> recreate your Conda environments on their machines it is sometimes a good practice to always explicitly 
-> specify the version number for each package that you install into an environment. If you are not 
+> recreate your Conda environments on their machines it is sometimes a good practice to explicitly 
+> specify the version number for each package that you install into an environment. 
+> 
+> Many packages use [semantic versioning](https://semver.org/)
```
Comment (ns-rse) [inline _episodes/02-working-with-environments.md:109]: :+1: useful explanation.

File: _episodes/02-working-with-environments.md [hunk 8]
```diff
@@ -6,23 +6,23 @@ questions:
 - "What is a Conda environment?"
 - "How do I create (delete) an environment?"
 - "How do I activate (deactivate) an environment?"
-- "How do I install packages into existing environments using Conda (+pip)?"
+- "How do I install packages into existing environments using Conda?"
```
Comment (ns-rse) [inline _episodes/02-working-with-environments.md:9]: Good catch, makes sense to mention it until introduced.

**Review by ns-rse — COMMENTED** [review]
All very sensible changes @bobturneruk I think its good to simplify and keep it focused rather than touching on things that people are unlikely to use.

I'll have a go at building the site locally and read through it more carefully this week.

File: _episodes/01-getting-started-with-conda.md (hunk 1 above)
Comment (bobturneruk) [reply _episodes/01-getting-started-with-conda.md]: Good point.

**Pull request merged**